pkg/types/baremetal/validation/platform.go

```diff
@@ -393,6 +397,17 @@ func ValidateProvisioning(p *baremetal.Platform, n *types.Networking, fldPath *f
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("libvirtURI"), p.LibvirtURI, err.Error()))
 	}
 
+	if p.BootstrapOSImage != "" {
+		if err := validateOSImageURI(p.BootstrapOSImage); err != nil {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("bootstrapOSImage"), p.BootstrapOSImage, err.Error()))
+		}
+	}
+	if p.ClusterOSImage != "" {
+		if err := validateOSImageURI(p.ClusterOSImage); err != nil {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("clusterOSImage"), p.ClusterOSImage, err.Error()))
+		}
+	}
+
```

Comment:
I don't think this is needed, the validate method gets called via the url annotation.

Reply:
The `validateOSImages` (just below) apply the rule for the field annotated with `osimageuri`

Reply:
@imain This still needs to go, this is already handled by the validateOSImages call just below this

Reply:
Ah crickey, sorry folks missed this.